contrib/flake.nix

```diff
@@ -75,26 +68,22 @@
           ps.msgpack
           ps.flake8 # for 'make pylint'
         ]);
-      in
-      rec {
-
+      in {
         packages = with pkgs; {
           default = neovim;
           inherit neovim neovim-debug neovim-developer;
         };
 
         checks = {
-          pylint = pkgs.runCommandNoCC "pylint"
-            {
-              nativeBuildInputs = [ pythonEnv ];
-              preferLocalBuild = true;
-            } "make -C ${./..} pylint > $out";
-
-          shlint = pkgs.runCommandNoCC "shlint"
-            {
-              nativeBuildInputs = [ pkgs.shellcheck ];
-              preferLocalBuild = true;
-            } "make -C ${./..} shlint > $out";
+          pylint = pkgs.runCommandNoCC "pylint" {
```

Comment:
in recent nixpkgs: `runCommandNoCC = throw "'runCommandNoCC' has been renamed to/replaced by 'runCommand'"; # Converted to throw 2022-09-24`  use runCommand instead

Reply:
> no rule to make target `shlint`

Changed it to runCommand though

Reply:
you've done well enough already.